src/category_theory/limits/limits.lean

```diff
@@ -872,6 +872,17 @@ begin
   simp,
 end
 
+@[simp]
+lemma has_limit.iso_of_equivalence_inv_π {F : J ⥤ C} [has_limit F] {G : K ⥤ C} [has_limit G]
+   (e : J ≌ K) (w : e.functor ⋙ G ≅ F) (j : J) :
+  (has_limit.iso_of_equivalence e w).inv ≫ limit.π F j =
+    limit.π G (e.functor.obj j) ≫ w.hom.app j :=
+begin
+  simp [has_limit.iso_of_equivalence, is_limit.cone_points_iso_of_equivalence_hom],
```

Comment:
nonterminal simp

Reply:
I'd argue that per https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.233638/near/209163865 this isn't actually a nonterminal simp, but in any case I've changed it!